### ISSUE TITLE
What coverage_baseline is supposed to do ?

### ISSUE THREAD

**Issue by alkuzad** [danmayer/coverband#71]
Readme mentions that it has something to do with initialization of app but it does not help. Why it is json and it's loaded on startup ?

**Comment by danmayer** [comment]
It is recording code usage, during app startup. If you don't do this you never run method definitions and other lines that are interpreted at app boot. It is recorded in JSON as you only need to do it once per code version.... Opposed to the runtime code analysis, this has confused some folks so it might be good to store it to the same data store through the adapter as anything else. That could be more flexible long term, so I can look at changing that.

basically you want to always record your baseline once and then record live stats ON TOP of the baseline. Thought if this was explained well if so I could try to update in the readme.

**Comment by alkuzad** [comment]
@danmayer thanks, now I think I've got it. I think this is just bad naming as "baseline" is not as clear as "initialization.

Initialization code is often not something we can have impact on but it's definitely worth to have coverage for it.

**Comment by danmayer** [comment]
gotcha, yeah not the best naming, I will try to improve the readme a bit.

**Comment by danmayer** [comment]
updated readme, closing this
